modules/admin_manual/pages/troubleshooting/path_filename_length.adoc

```diff
@@ -0,0 +1,102 @@
+:getconf-url: http://manpages.ubuntu.com/manpages/focal/man1/getconf.1.html
+
+== Introduction
+
+Depending on the underlaying filesystem of a mount point, the maximum length of a path component and the filename can differ. This is important if you start copying or moving single files or even complete paths from one mount to another where the target mount has a more restrictive lengt rule than the source. This can also be an issue when using a synchronisation client when using an Operating System (OS) having a different filesystem than the source mount filesystem has. The following table should give you a brief overview as guideline.
+
+== Limitations
+
+See the {fs-limits-url}[Comparison of file systems] for in depth details on various filesystem path and filname limitations.
+
+NOTE: While a filesystem can hande the limits as described in the table below, applications like Explorer, Finder, the Shell or other apps may get issues handling these limits. See the special notes below the table.
+
+NOTE: The ownCloud database has a size limit storing a path/file string with 4000 bytes. This must not be exceeded.
+
+[caption=]
+.Filename and Path Length Limitations 
+[cols="30%,80%,80%",options="header",]
+|===
+| Filesystem
+| max. Path Length
+| max. Filename Length
+
+| (*) Btrfs
+| No limit defined
+| 255 bytes
+
+| (*) ext2
+| No limit defined
+| 255 bytes
+
+| (*) ext3
+| No limit defined
+| 255 bytes
+
+| (*) ext4
+| No limit defined
+| 255 bytes
+
+| (*) XFS
+| No limit defined
+| 255 bytes
+
+| (*) ZFS
+| No limit defined
+| 255 bytes
+
+| APFS
+| Unknown (**)
+| 255 UTF-8 characters
+
+| FAT32
+a| 32,760 Unicode characters with *each* path component no more than 255 characters
+| 8.3 (255 UCS-2 code units with VFAT LFNs)
+
+| exFAT
+a| 32,760 Unicode characters with *each* path component no more than 255 characters
+| 255 UTF-16 characters
+
+| NTFS
+a| 32,767 Unicode characters with *each* path component (directory or filename) up to 255 characters long (MAX_PATH).
+
+====
+Starting in Windows 10, version 1607, MAX_PATH limitations have been removed from common Win32 file and directory functions. However, you must opt-in to the new behavior. For more details see {enable-long-paths-url}[Enable Long Paths in Windows 10, Version 1607, and Later]
+====
+| 255 characters
+|===
+
+(*)::
+In Unix environments, PATH_MAX with 4096 bytes and NAME_MAX with 255 bytes are very common limitations for applications including the Shell. You can get the current limitations by typing following example commands, see the {getconf-url}[getconf manpage] for details:
++
+[source,console]
+----
+getconf NAME_MAX /
+255
+----
++
+[source,console]
+----
+getconf PATH_MAX /
+4096
+----
+
+(**)::
+Even not officially documented, but when searching on the internet there is a limit when having path names exceeding 1024 bytes. Users report warnings in Finder, the Shell or apps about this behaviour. This can be proofed with:
```

Comment:
```suggestion
Although not officially documented, when searching on the internet there is a limit with path names exceeding 1024 bytes. Users report warnings in Finder, the Shell or apps about this behavior. This can be verified with:
```